File: Programmers/Level_1/크기가 작은 부분문자열.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(string t, string p)
{
    int answer = 0;
    long long p_num = stoll(p);

    for (int i = 0; i <= t.size() - p.size(); i++)
    {
        string temp = "";
        for (int j = i; j < i + p.size(); j++)
        {
            temp += t[j];
        }
        if (stoll(temp) <= p_num)
        {
            answer++;
        }
    }

    return answer;
}

File: Programmers/Level_1/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    var cnt = 0;

    for (var i = 0; i <= t.length - p.length; i++) {
        var temp = "";
        for (var j = i; j < i + p.length; j++) {
            temp += t[j];
        }
        console.log(Number(temp));
        if (Number(temp) <= Number(p)) {
            answer += 1;
        }
    }
    console.log(cnt);
    return answer;
}
